retrieval/series_cache_test.go

```diff
@@ -453,6 +462,75 @@ func TestSeriesCache_Filter(t *testing.T) {
 	}
 }
 
+func TestSeriesCache_CounterAggregator(t *testing.T) {
+	resourceMaps := []ResourceMap{
+		{
+			Type:     "resource2",
+			LabelMap: map[string]labelTranslation{"__resource_a": constValue("resource_a")},
+		},
+	}
+	// Populate the getters with data.
+	targetMap := targetMap{
+		"job1/inst1": &targets.Target{
+			Labels:           promlabels.FromStrings("job", "job1", "instance", "inst1"),
+			DiscoveredLabels: promlabels.FromStrings("__resource_a", "resource2_a"),
+		},
+	}
+	metadataMap := metadataMap{
+		"job1/inst1/metric1": &scrape.MetricMetadata{Type: textparse.MetricTypeGauge, Metric: "metric1"},
+	}
+	logger := log.NewNopLogger()
+	aggr, _ := NewCounterAggregator(logger, &CounterAggregatorConfig{
+		"counter1": &CounterAggregatorMetricConfig{Matchers: [][]*promlabels.Matcher{
+			{&promlabels.Matcher{Type: promlabels.MatchEqual, Name: "a", Value: "a1"}},
+		}},
+	})
+	c := newSeriesCache(logger, "", [][]*promlabels.Matcher{
+		{&promlabels.Matcher{Type: promlabels.MatchEqual, Name: "b", Value: "b1"}},
+	}, nil, targetMap, metadataMap, resourceMaps, "", false, aggr)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	for idx, tt := range []struct {
+		name         string
+		lset         labels.Labels
+		wantExported bool
+		wantTracked  bool
```

Comment:
I know `exported` and `tracked` are well defined in the code, but it'd be helpful to have some context here to understand the test. Can you add a one-line comment to each?

Reply:
Done.